src/LinuxTracing/PerfEvent.h

```diff
@@ -201,6 +203,8 @@ using SchedSwitchPerfEvent = TypedPerfEvent<SchedSwitchPerfEventData>;
 
 struct SchedWakeupPerfEventData {
   pid_t woken_tid;
+  pid_t was_blocked_by_thread;
+  pid_t was_blocked_by_process;  // Could be zero if we don't have access to the process id.
```

Comment:
This is nitpicky, but while `was_blocked_by` is a good name for the thread-state-related structs and protos, here we are still at the lower level of the raw tracepoint. In fact, the tracepoint is actually originated by the "was blocked by" thread. I think I would prefer `waker_tid`/`waker_pid` (in contrast with `woken_tid`).